### ISSUE TITLE
Add HTML lang attribute

### ISSUE THREAD

**Issue by Crustan** [Crustan/downunder#5]
`<html>` tag should have a `lang` attribute to improve accessibility.

From Lighthouse:

<img width="738" alt="image" src="https://github.com/Crustan/downunder/assets/9121571/78059e51-92b9-42f8-bd53-e227f962fb04">
